src/Pages/kcy/Main/MainKim.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Navbar from "./Navbar";
+import Feed from "./Feed";
+import MainRight from "./MainRight";
+import Footer from "./Footer";
+import "../config/Main.scss";
+import "../config/Reset.scss";
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
```suggestion
import "../config/Reset.scss";

class Main extends React.Component {
```
- 구분되는 요소들 끼리는 한줄 띄워 분리해주세요
- 불필요한 constructor입니다